recipes/pdoc/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pdoc" %}
+{% set version = "11.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mitmproxy/pdoc/archive/refs/tags/v{{ version }}.tar.gz
+  # pdoc does not currently upload a distrobution (tar.gz) file to PyPi, so I'm
+  # stuck using the github URL for now. I requested this to be fixed, and it is
+  # accepted, I can switch to the follow url:
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # Until then, I must manually download and check with "openssl sha256 <download file>"
+  sha256: 233f9f34d4f7a2c8a14ca7b2f411bdcebfc1541def352057410d94d6f2badcb0
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+      - pdoc = pdoc.__main__:cli
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
You cannot be noarch and require a compiler. Can you check if you have compiled extensions?